portfolio-project-matching-app/backend/dao.js

```diff
@@ -224,6 +226,51 @@ const getPayload = (coll, id1, id2) => {
 
 // see daoProject.js, daoUser.js, and daoTechnology.js
 
+const readAllDocs = async (coll) => {
+    /*
+    DESCRIPTION:    retrieves all documents in the specified collection and
+                    returns an array of custom objects. Note, the associations
+                    will not be populated in the objects.
+
+    INPUT:          coll (string): collection to get documents from
+
+    RETURN:         array of custom objects
+    */
```

Comment:
Quick and dirty generic function for READING all docs from a collection. Note, this is restricted to 'projects', 'users', and 'technologies'.